src/components/drag-and-drop/drag-and-drop.stories.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import notes from './notes.md';
+import { DraggableContext } from './drag-and-drop';
+import {
+  Table,
+  TableHeader,
+  TableRow,
+  TableCell
+} from '../table';
+
+const store = new Store({
+  dndData: [{
+    id: '0',
+    name: 'UK'
+  }, {
+    id: '1',
+    name: 'Germany'
+  }, {
+    id: '2',
+    name: 'China'
+  }, {
+    id: '3',
+    name: 'US'
+  }]
+});
+
+const onDrag = (originalIndex, newIndex) => {
+  action('drag')();
+  const array = store.get('dndData');
```

Comment:
Maybe can we find a better name for it than `array`.

Reply:
👍